src/core/Solana/sendAndConfirmTransaction.ts

```diff
@@ -41,71 +42,86 @@ export async function sendAndConfirmTransaction(
     preflightCommitment: 'confirmed',
   }
 
-  for (const connection of connections) {
-    connection
-      .sendRawTransaction(signedTxSerialized, rawTransactionOptions)
-      .catch()
-  }
-
-  const abortControllers: AbortController[] = []
-
-  const confirmPromises = connections.map(async (connection) => {
-    const abortController = new AbortController()
-    abortControllers.push(abortController)
-    try {
-      const blockhashResult = await connection.getLatestBlockhash('confirmed')
-
-      const confirmTransactionPromise = connection
-        .confirmTransaction(
-          {
-            signature: txSignature,
-            blockhash: blockhashResult.blockhash,
-            lastValidBlockHeight: blockhashResult.lastValidBlockHeight,
-            abortSignal: abortController.signal,
-          },
-          'confirmed'
-        )
-        .then((result) => result.value)
-
-      let signatureResult: SignatureResult | null = null
-      let blockHeight = await connection.getBlockHeight('confirmed')
+  const pollPromises = connections.map(async (connection) => {
+    const timeout = 60000
```

Comment:
It would be nice to keep the previous logic and keep it based on the block hash expiration, instead of custom timeouts.